src/client/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+const App = () => {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>Welcome to Survey Shark!</h1>
```

Comment:
```suggestion
        <h1>Welcome to SurveyShark!</h1>
```
Since I thought we agreed the name has no space in it.

Reply:
works for me, fixed!

Reply:
yes I agree about testing along the way. I'd suggest that if a developer creates a component/view they should at least make sure its accessible and there are unit/integration tests.